Cargo.toml

```diff
@@ -31,4 +31,6 @@ rand = { version = "0.7", default-features = false, features = ["alloc"], option
 slab = "0.4"
 
 [dev-dependencies]
+quickcheck = "0.9"
+quickcheck_macros = "0.9"
```

Comment:
It seemed from the docs that the macro is the "old way" and the attribute is the "new way". If that's not true or if we want to use the macro anyway, lemme know!

Reply:
I honestly don't know :sweat_smile: